rfc6962-bis.xml

```diff
@@ -1363,6 +1363,14 @@ entry specified by <spanx style="verb">start</spanx>.
           Provided that a TLS client includes the <spanx style="verb">transparency_info</spanx> extension type in the ClientHello, the TLS server SHOULD include the <spanx style="verb">transparency_info</spanx> extension in the ServerHello with <spanx style="verb">extension_data</spanx> set to a <spanx style="verb">TransItemList</spanx>. The TLS server SHOULD ignore any <spanx style="verb">extension_data</spanx> sent by the TLS client. Additionally, the TLS server MUST NOT process or include this extension when a TLS session is resumed, since session resumption uses the original session information.
         </t>
       </section>
+      <section title="cached_info TLS Extension">
+        <t>
+          Provided that a TLS client includes the <spanx style="verb">cached_info</spanx> (<xref target="RFC7924"/>) extension type in the ClientHello with a <spanx style="verb">CachedObject</spanx> of type <spanx style="verb">ct_compliant</spanx> (see <xref target="tls_cachedinfo_extension"/>) and at least one <spanx style="verb">CachedObject</spanx> of type <spanx style="verb">cert</spanx>, and provided that the TLS server then sends a modified Certificate message (as described in section 4.1 of <xref target="RFC7924"/>) and includes the <spanx style="verb">transparency_info</spanx> extension in the ServerHello, the TLS server SHOULD NOT include any <spanx style="verb">TransItem</spanx> structures of type <spanx style="verb">x509_sct_with_proof_v2</spanx>, <spanx style="verb">x509_sct_v2</spanx>, <spanx style="verb">precert_sct_with_proof_v2</spanx> or <spanx style="verb">precert_sct_v2</spanx> in the <spanx style="verb">TransItemList</spanx> that it includes in the <spanx style="verb">transparency_info</spanx> extension.
```

Comment:
To be clear, does the TLS client have to also send the transparency_info extension  for this to have effect?